### DISCUSSION TITLE
magick: font substitution required `Courier' @ error/type.c/GetTypeInfoByFamily/495.

### DISCUSSION THREAD

**Discussion by luojc0517** [ImageMagick/ImageMagick#1904] [General]
when I tried to convert .wmf file to png on MacOS Mojave, I got this font error
```
#sudo magick example.wmf example_magick.png

Fontconfig warning: ignoring UTF-8: not a valid region tag
magick: font substitution required `Courier' @ error/type.c/GetTypeInfoByFamily/495.
```

**Comment by urban-warrior** [comment]
Your example.wmf image likely requires a Courier font and its not available on your system.  Type `identify -list font` for a list of available fonts that ImageMagick discovered.  Our system includes Courier fonts:

```
  Font: Courier-Bold
    family: Courier
    style: Normal
    stretch: Normal
    weight: 700
    glyphs: /usr/share/fonts/urw-base35/NimbusMonoPS-Bold.t1

```

**Reply by CxcPDF** [reply]
hi, did you have solved this problem? I meet the same problem
